src/components/ConversationSettings/ListableSettings.vue

```diff
@@ -85,12 +92,28 @@ export default {
 		conversation() {
 			return this.$store.getters.conversation(this.token) || this.$store.getters.dummyConversation
 		},
+
+		summaryLabel() {
+			switch (this.listable) {
+			case CONVERSATION.LISTABLE.ALL:
+				return t('spreed', 'Conversation is open to registered and guest app users')
+			case CONVERSATION.LISTABLE.USERS:
+				return t('spreed', 'Conversation is open to registered users')
```

Comment:
Then, I would suggest to change to sthg like this :p 
```suggestion
				return t('spreed', 'Conversation is open to registered users and guests joined from the sharing link')
```

Reply:
Listable settings are not related to simple guests, only Guests app